db/seedData.js

```diff
@@ -51,18 +51,25 @@ const {bakeryProductsToCreate} = require('./productSeedArrays/bakerySeed.js')
       "subCategory" VARCHAR(255) NOT NULL,
       "imageURL" VARCHAR(255) NOT NULL
     );
-  
+
     CREATE TABLE orders (
       id SERIAL PRIMARY KEY,
       // todo "sessionId" INTEGER, not sure how this would work. Will circle back if time allows
       "userId" INTEGER REFERENCES users(id),
-      "productId" INTEGER REFERENCES products(id),
-      quantity INTEGER NOT NULL,
+      "checkoutDate
       "isCheckedOut" BOOLEAN DEFAULT false, 
-      "checkoutPrice" DECIMAL (10,2), 
-      UNIQUE ("userId", "productId")
+      "checkoutSum" DECIMAL (10,2), 
```

Comment:
```suggestion
      "checkoutSum" DECIMAL (10,2)
```

I believe SQL doesn't allow trailing commas (commas at the end of the last line)